app/src/main/java/fr/free/nrw/commons/customselector/ui/selector/ImageFragment.kt

```diff
@@ -234,6 +252,24 @@ class ImageFragment :
         return binding?.root
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        viewLifecycleOwner.lifecycleScope.launch {
```

Comment:
Could you add a 1-line comment explaining from a user's point of view what this does? High-level.